src/components/Select.vue

```diff
@@ -345,16 +349,17 @@
               :id="inputId"
               aria-label="Search for option"
       >
-
-      <button 
-        v-show="showClearButton" 
-        :disabled="disabled" 
+      <button
+        v-show="showClearButton"
+        :disabled="disabled"
         @click="clearSelection"
-        type="button" 
-        class="clear" 
-        title="Clear selection" 
+        type="button"
+        class="clear"
+        title="Clear selection"
       >
-        <span aria-hidden="true">&times;</span>
+        <slot name="clear">
+          <span class="cross" aria-hidden="true">&times;</span>
```

Comment:
Is the `class=“cross”` is needed here? Or perhaps a more explicit name for styling would be better?

Reply:
maybe it will more scalable to have a slot for the `vs__actions` itself